src/main/java/games/strategy/engine/framework/ArgParser.java

```diff
@@ -0,0 +1,58 @@
+package games.strategy.engine.framework;
+
+import java.util.Arrays;
+
+import games.strategy.triplea.settings.SystemPreferenceKey;
+import games.strategy.triplea.settings.SystemPreferences;
+
+public class ArgParser {
+  /**
+   * Move command line arguments to System.properties
+   * @return Return true if all args were valid and accepted, false otherwise.
+   */
+  public static boolean handleCommandLineArgs(
```

Comment:
This is not new code, largely moved without modification from:

https://github.com/triplea-game/triplea/pull/1831/files#diff-fbe5bf15209a9e3d2ec3c941c7fb72b0L221

and
https://github.com/triplea-game/triplea/pull/1831/files#diff-fbe5bf15209a9e3d2ec3c941c7fb72b0L338

Since the main part of `handleCommandLineArgs` was done inline with other methods, it had no concept previously of a return value. So for now I decided to go with a boolean to represent success/fail case. I'm not very happy with this, but also it's something I'd like to improve in further iterations.